README.md

```diff
@@ -65,15 +65,15 @@ In order to use COSI, it requires a known NORAD ID of an active satellite that i
 
 **Get Latest TLE for Bobcat-1:** *(NORAD ID: 46922)*
 
-`$` `uniclogs-cosi --tle 46922`
+`$` `uniclogs-cossi --tle 46922`
 
 **Get Satellite Metadata and telemetry for Bobcat-1:**
 
-`$` `uniclogs-cosi --satellite --telemetry 46922`
+`$` `uniclogs-cossi --satellite-metadata --telemetry 46922`
```

Comment:
@dmitri-mcguckin should this be `satellite-metadata`? just `satellite` didn't work